TheMET/TheMET/Network/NetworkManager.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  NetworkManager.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 21/03/2023.
+//
+
+import Foundation
+
+class NetworkManager {
+    
+    func loadData(urlString: String, parameters: [String : String], completion: @escaping([String: Any]?) -> Void)  {
+        var components = URLComponents(string: urlString)!
+        components.queryItems = parameters.map{ (key, value) in
+            URLQueryItem(name: key, value: value)
+        }
+        components.percentEncodedQuery = components.percentEncodedQuery?.replacingOccurrences(of: "+", with: "%2B")
+        let urlRequest = URLRequest(url: components.url!)
```

Comment:
Do not force unwrap `components.url!`. Instead, unwrap it properly and execute `completion` like you do for an error.

Reply:
Fixed